website/content/en/status/report-2023-07-2023-09/portmgr.adoc

```diff
@@ -3,40 +3,47 @@
 Links: +
 link:https://www.FreeBSD.org/ports/[About FreeBSD Ports] URL:link:https://www.FreeBSD.org/ports/[] +
```

Comment:
@bapt @mat813 @pizzamig @rene0 @tcberner (Moinur Rahman not in GitHub, I guess)

**About FreeBSD Ports** no longer exists. 

For status report purposes, please: 

* which alternative page will portmgr prefer?

Reply:
<https://ports.freebsd.org/cgi/ports.cgi?stype=faq> includes a link out to <https://docs.freebsd.org/en/books/handbook/ports/#ports-using>, so maybe **Using the Ports Collection** should supersede **About FreeBSD Ports**.

Thoughts? 

TIA

Reply:
Unless someone from portmgr replies, please leave the link as it is. It might also be that the page is planned to be changed in some near future.